File: functions/InLocCIIRC_utils/buildCutoutName/buildCutoutName.py
import os
from pathlib import Path

def buildCutoutName(cutoutPath, extension):
    basename = Path(cutoutPath).stem
    spaceName = cutoutPath.split('/')[0]
    splitBasename = basename.split('_')
    name = splitBasename[0] + '_' + spaceName + '_' + '_'.join(splitBasename[1:]) + extension
    return name

File: functions/InLocCIIRC_utils/buildCutoutName/buildCutoutName.m
function name = buildCutoutName(cutoutPath, extension)
    [~, basename, ~] = fileparts(cutoutPath);
    spaceName = strsplit(cutoutPath, '/');
    spaceName = spaceName{1};
    splitBasename = strsplit(basename, '_');
    name = [splitBasename{1}, '_', spaceName, '_', strjoin(splitBasename(2:end), '_'), extension];
end